doc/source/advanced/datatype.rst

```diff
@@ -0,0 +1,90 @@
+***********************************
+Data Types in Lux
+***********************************
+
+.. note:: You can follow along this tutorial in a Jupyter notebook. 
+
+Currently, Lux supports four data types:
+
+* Quantitative
+* Temporal
+* Nominal
+* ID
+
+Note that these data types are different from Pandas `dtype`. These data types are automatically inferred based on the input `dataframe`.
+The following code snippet shows what data types were inferred:
+
+.. code-block:: python
+
+  df = pd.read_csv("https://github.com/lux-org/lux-datasets/blob/master/data/college.csv?raw=true")
+  df.data_type
+
+
+Quantitative
+------------
+The quantitative data type is used when there is a count or measure of a certain attribute. 
+In the example above, the column `AcceptanceRate` is quantitative because it is a measure. 
+Also, any aggregate such as means and medians will be categorized as quantitative. 
+Usually, Lux is able to compute Correlations between two quantitative columns or show the distribution of each column.
```

Comment:
Section looks good! Maybe you could also mention how correlations are shown via scatterplots/heatmaps and distributions via histograms